packit_service/models.py

```diff
@@ -440,6 +442,27 @@ def get_all(cls) -> Optional[Iterable["CoprBuildModel"]]:
         with get_sa_session() as session:
             return session.query(CoprBuildModel).order_by(desc(CoprBuildModel.id)).all()
 
+    @classmethod
+    def get_merged_chroots(
+        cls, first: int, last: int
+    ) -> Optional[Iterable["CoprBuildModel"]]:
+        with get_sa_session() as session:
+            builds_list = (
```

Comment:
Keep naming simple :)

```suggestion
            builds = (
```